HeterogeneousCore/SonicTriton/scripts/cmsTriton

```diff
@@ -346,10 +346,22 @@ edit_model(){
 	IFS='/' read -ra ADDR <<< "$MODELNAME"
 	CONFIG=$TMPDIR/$LOCALMODELREPO/${ADDR[-1]}/config.pbtxt
 	
+	MODELLOC=$(echo ""${ADDR[@]:0:${#ADDR[@]}-1} | sed "s/ /\//g")
```

Comment:
this could be moved inside the ensemble case, since it's only used there

Reply:
Yes, I moved it